### PR TITLE
CD staff notifs

### PR DESCRIPTION
**Pull request by H4ckerJ4cker**
Added a notification when a cd staff member comes online.

### CODE REVIEW

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -151,6 +152,13 @@ async def on_ready(self):
 
         await self.migrate_quotes()
 
+    @Cog.listener()
+    async def on_member_update(self, before, after):
+        if before.status == Status.offline and after.status == Status.online:
+            if any(i.id == STAFF_ROLE_ID for i in after.roles):
+                alert_channel = self.bot.get_channel(LOGGING_CHANNEL_ID)
+                await alert_channel.send("A CD staff member came online :eyes:")
```
Comment (fractalpenguin) [inline cdbot/cogs/fun.py:160]: It might be useful to have the message say which staff member came online i.e include the username of who came online.

Comment (H4ckerJ4cker) [reply cdbot/cogs/fun.py]: This functionality has been added.

**Comment by thebeanogamer** [comment]
This... isn't useful.

**Comment by H4ckerJ4cker** [comment]
You asked for it in vc once @thebeanogamer.

**Pull request closed without merging**